io-engine/src/lvs/lvs_store.rs

```diff
@@ -874,11 +874,40 @@ impl Lvs {
         Ok(())
     }
 
-    /// Grows the online (live) pool.
+    /// Rescans the bdev and triggers live LVS grow i.e. without closing the blobs and unloading the blobstore.
     #[tracing::instrument(level = "debug", err)]
     pub async fn grow(&self) -> Result<(), LvsError> {
         info!("{self:?}: growing lvs...");
 
+        let cname = self.base_bdev().name().into_cstring();
+        let uri_str = &self.base_bdev().bdev_uri_str().unwrap_or_default();
```

Comment:
`unwrap_or_default`, what is the default here? Should we not exit rather than continuing with default?

Reply:
It would probably exit from next step?

Reply:
Yes, Cant parse so exit there..